v2/common/src/main/java/com/google/cloud/teleport/v2/utils/BeamSchemaUtils.java

```diff
@@ -100,17 +100,27 @@ private static Schema fromJson(JsonParser jsonParser) throws IOException, Schema
   private static List<Field> getFieldsfromJsonNode(JsonNode jsonNode)
       throws SchemaParseException {
     List<Field> fields = new LinkedList<>();
+    Field field;
     for (JsonNode node : jsonNode) {
       if (!node.isObject()) {
         throw new SchemaParseException("Node must be object: " + node.toString());
       }
       String type = getText(node, FIELD_TYPE, "type is missed");
       String name = getText(node, FIELD_NAME, "name is missed");
-      fields.add(Field.of(name, stringToFieldType(type)));
+      boolean nullable = getOptionalBoolean(node, FIELD_NULLABLE, false);
+      field = nullable ? Field.nullable(name, stringToFieldType(type))
+          : Field.of(name, stringToFieldType(type));
+      fields.add(field);
     }
     return fields;
   }
 
+
+  private static boolean getOptionalBoolean(JsonNode node, String key, boolean defaultValue) {
```

Comment:
not sure that I understand naming. You planning to use this method to parse types of fields in the schema? 
if not let's name it something like "isOptionalField"

Reply:
any other cases where we need to read boolean values from json?

Reply:
At this moment there are no other cases.

Reply:
Let's keep it, I think this way the code looks a bit more clear